app/src/main/java/de/xikolo/viewmodels/helpdesk/TicketViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package de.xikolo.viewmodels.helpdesk
+
+import de.xikolo.models.TicketTopic
+import de.xikolo.models.dao.CourseDao
+import de.xikolo.network.jobs.CreateTicketJob
+import de.xikolo.viewmodels.base.BaseViewModel
+import de.xikolo.viewmodels.shared.CourseListDelegate
+
+class TicketViewModel : BaseViewModel() {
+
+    private val coursesDao = CourseDao(realm)
+    private val courseListDelegate = CourseListDelegate(realm)
+
+    val courses = coursesDao.all()
```

Comment:
Can be replaced by `val courses = courseListDelegate.courses`. This removes the `CourseDao` dependency.